versioned/storage/bigtable/src/main/java/org/projectnessie/versioned/storage/bigtable/BigTablePersist.java

```diff
@@ -625,7 +686,10 @@ protected Obj computeNext() {
 
         List<RowCell> objCells = row.getCells(FAMILY_OBJS, QUALIFIER_OBJS);
         ByteBuffer obj = objCells.get(0).getValue().asReadOnlyByteBuffer();
-        return deserializeObj(id, obj);
+        List<RowCell> objVersionCells = row.getCells(FAMILY_OBJS, QUALIFIER_OBJ_VERS);
+        String versionToken =
```

Comment:
Maybe extract to a method? Repeated 3 times.

Reply:
done